packages/client/hmi-client/src/components/workflow/ops/funman/tera-funman-drilldown.vue

```diff
@@ -559,6 +718,178 @@ watch(
+	}
+}
+
+function updateSelectedStates(event) {
+	emit('update-state', {
+		...props.node.state,
+		chartSettings: updateChartSettingsBySelectedVariables(
+			chartSettings.value,
+			ChartSettingType.VARIABLE_COMPARISON,
+			event.selectedVariable
+		)
+	});
+}
+
+function updateSelectedParameters(event) {
+	emit('update-state', {
+		...props.node.state,
+		chartSettings: updateChartSettingsBySelectedVariables(
+			chartSettings.value,
+			ChartSettingType.DISTRIBUTION_COMPARISON,
+			event.selectedVariable
+		)
+	});
+	updateParameterCharts(); // Rerender when we remove/add charts since they are all contained in the same visualization
+}
+
+function removeChartSetting(chartId: string) {
+	const chartType = chartSettings.value.find((setting) => setting.id === chartId)?.type;
+	emit('update-state', {
+		...props.node.state,
+		chartSettings: removeChartSettingById(chartSettings.value, chartId)
+	});
+	if (chartType === ChartSettingType.DISTRIBUTION_COMPARISON) {
+		updateParameterCharts();
+	}
+}
+
+async function renderCharts() {
+	processedFunmanResult = processFunman(funmanResult, onlyShowLatestResults.value);
+
+	updateStateCharts();
+	updateParameterCharts();
+
+	// For displaying model/model configuration
+	// Model will be the same on runId change, no need to fetch it again
+	if (!model.value) {
+		model.value = await getModelByModelConfigurationId(funmanResult.modelConfiguration.id);
+		if (!model.value) {
+			logger.error('Failed to fetch model');
+			return;
+		}
+		const response = await getMMT(model.value);
+		if (response) {
+			mmt = response.mmt;
+			mmtParams.value = response.template_params;
+		}
+	}
+
+	configuredMmt.value = makeConfiguredMMT(mmt, funmanResult.modelConfiguration);
+	calibratedConfigObservables.value = funmanResult.modelConfiguration.observableSemanticList.map(
+		({ referenceId, states, expression }) => ({
+			id: referenceId,
+			name: referenceId,
+			states,
+			expression
+		})
+	);
+}
+
+watch(
+	() => props.node.state.runId,
+	async () => {
+		if (!props.node.state.runId) return;
+		const rawFunmanResult = await getRunResult(props.node.state.runId, 'validation.json');
+		if (!rawFunmanResult) {
+			logger.error('Failed to fetch funman result');
+			return;
+		}
+		notebookText.value = rawFunmanResult;
+		funmanResult = JSON.parse(rawFunmanResult);
+		constraintsResponse = funmanResult.request.constraints;
+		validatedModelConfiguration.value = funmanResult.modelConfiguration;
+
+		stateOptions.value = funmanResult.model.petrinet.model.states.map(({ id }) => id);
+		parameterOptions.value = funmanResult.request.parameters
+			.filter((d: any) => d.interval.lb !== d.interval.ub)
+			.map(({ name }) => name);
+
+		// Initialize default output settings
+		const state = _.cloneDeep(props.node.state);
+		state.chartSettings = updateChartSettingsBySelectedVariables(
+			[],
+			ChartSettingType.VARIABLE_COMPARISON,
```

Comment:
```suggestion
			ChartSettingType.VARIABLE,
```
I think it should be VARIABLE type, if you are dealing with single state variable in the chart. 
`VARIABLE_COMPARISON` should be used for comparing multiple variables in the same chart. 
I miss used it in the calibrate drilldown and just fixed it recently as well :)